modin/core/dataframe/algebra/default2pandas/default.py

```diff
@@ -68,7 +68,7 @@ def call(cls, func, obj_type=pandas.DataFrame, inplace=None, fn_name=None):
         if type(fn) == property:
             fn = cls.build_property_wrapper(fn)
 
-        def applyier(df, *args, **kwargs):
+        def applyier(df, *args, dtypes=None, **kwargs):
```

Comment:
this is not used here, should we add an assert that it's always `None`?

Reply:
There is a generic code that sets this argument even for the pure pandas case, so it's not always `None`.

Reply:
Is this dtypes used anywhere at all? Can you give an example?

Reply:
It's used for example here: `df > 6` (code path will be through `map` function)